lib/vdsm/storage/sp.py

```diff
@@ -1481,9 +1481,12 @@ def extendVolume(self, sdUUID, volumeUUID, size):
 
         # Extend volume without refreshing its size. If the SPM host see the
         # new size immediately after extension, this can cause data corruption
-        # during VM migration when the source host is SPM. Volume size will be
-        # refreshed in Vm.after_volume_extension(), which is a callback of disk
-        # extend command.
+        # during VM migration when the source host is SPM.
+        #
+        # The logical volume will be refreshed by the host requesting the
+        # extend when we send extend reply, after refreshing the disk on the
```

Comment:
"extend" as a noun?

Reply:
We use it extend like this see livemerge. We have EXTEND and COMMIT phases. This probably should be:

    ... requesting the extend when we send an extend reply ...

Reply:
OK

Reply:
Using new "extension" and "reply".